### ISSUE TITLE
generated output only does relative imports

### ISSUE THREAD

**Issue by a-morales** [stephenh/ts-proto#5]
If there's a `.proto` file that is not on a root path and it imports another `.proto` file then the resulting typescript file will have the import as a relative import causing typescript to throw errors. For example given the following file structure

```
- foo/
  - bar.proto
  - baz.proto
```

with `bar.proto` being

```proto
syntax = "proto3";

package foo;

import "google/protobuf/wrappers.proto";
import "foo/baz.proto";
...
```

The generated output of `ts-proto` would be
```
- google/
  - protobuf/
    - wrappers.ts
- foo/
  - bar.ts
  - baz.ts
```

With `bar.ts` being

```typescript
import * from "./google/protobuf/wrappers";
import * from "./foo/baz.ts";

...
```

Since a relative path is specified for imports, typescript throws errors that it can't find the modules. A proper fix is to either compute the relative paths for imports when generating the output or just have them be "top level" packages (e.g `import * from 'google/protobuf/wrappers'`) so that typescript can be configured to include the packages for module resolution.

**Comment by a-morales** [comment]
#3 is related to this.

**Comment by stephenh** [comment]
I'm 90% sure this is fixed in 1.4.1 / fixed while fixing #4. Going to close based on that guess, but let me know if its not.
